proto/celestia/core/v1/tx/tx.proto

```diff
@@ -33,6 +33,8 @@ message TxStatusResponse {
     // and returns whether it was successful or errored. A non zero
     // execution code indicated an error.
     uint32 execution_code = 3;
+    // error log for failed transactions.
+    string error = 4;
     // status is the status of the transaction.
-    string status = 4;
+    string status = 5;
```

Comment:
So whenever you modify the field number, that's a breaking change. To do this in a non-breaking change you should add `string error = 5;` not `4`

Reply:
we have never released this so we don't have to maintain any backwards compatibility